parser/src/main/java/log/parser/model/HeroDamage.java

```diff
@@ -0,0 +1,17 @@
+package log.parser.model;
+
+import javax.persistence.Entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Data;
+
+@Data
+@Entity (name="hero_damage")
```

Comment:
Nit: I would prefer using camelCasing for request/response payloads in a Java service as it  is more consistent with language naming conventions. snake_casing is normally used for python, php and ruby api's because of the same reasons.

Reply:
I updated table names by using camelCasing naming conventions.